spec/controllers/api/v2/searches_controller_spec.rb

```diff
@@ -365,6 +365,17 @@
       it { is_expected.to respond_with :success }
     end
 
+    context 'when the search options include sort_by' do
```

Comment:
This strikes me as an unnecessarily specific test. I'm thinking it could be something more like:
 ```
when the search options are valid
    it passes the correct options to its ApiI14ySearch object
```

 and test all the expected options.